src/api/app/views/webui/package/show.html.haml

```diff
@@ -26,16 +38,17 @@
                                                   revision: @revision,
                                                   project: @project,
                                                   package: @package }
-      = render partial: 'bottom_actions', locals: { bugowners_mail: @bugowners_mail,
-                                                    configuration: @configuration,
-                                                    revision: @revision,
-                                                    current_rev: @current_rev,
-                                                    spec_count: @spec_count,
-                                                    cleanup_source: @cleanup_source,
-                                                    services: @services,
-                                                    project: @project,
-                                                    package: @package,
-                                                    is_working: @forced_unexpand.blank? }
+        = unless flipper_responsive?
```

Comment:
This should be ` - unless flipper_responsive?`

Reply:
i have changed it